demos/smartwatch/lv_demo_smartwatch_home.h

```diff
@@ -0,0 +1,116 @@
+/**
+ * @file lv_demo_smartwatch_home.h
+ *
+ */
+
+#ifndef LV_DEMO_SMARTWATCH_HOME_H
+#define LV_DEMO_SMARTWATCH_HOME_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_demo_smartwatch.h"
+
+#if LV_USE_DEMO_SMARTWATCH
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ * GLOBAL PROTOTYPES
+ **********************/
+
+/**
+ * Create the home page. Called only once.
+ */
+void lv_demo_smartwatch_home_create(void);
+
+/**
+ * load the home screen
+ * @param anim_type    screen load animation to use
+ * @param time    animation time
+ * @param delay     delay time before loading the screen
+ */
```

Comment:
Please format these like this:
```suggestion
 * Load the home screen
 * @param anim_type    screen load animation to use
 * @param time         animation time
 * @param delay        delay time before loading the screen
 */
```